dev-packages/node-integration-tests/suites/anr/basic-session.js

```diff
@@ -10,7 +10,6 @@ setTimeout(() => {
 Sentry.init({
   dsn: 'https://public@dsn.ingest.sentry.io/1337',
   release: '1.0',
-  debug: true,
```

Comment:
I also removed a bunch of `debug: true` leftovers in the tests, to clean up the output.